PusherMainView/PusherMainView/PushData.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+struct PushData: Equatable {
```

Comment:
`PushData` looks convenient to use but here we mix APNS model and FCM model.
Is it possible to separate APNS model and FCM model?

Reply:
Yeah that's a good idea.
Take a look if the solution I came up with is fair enough in the most recent commit